en/services/parachute.md

```diff
@@ -0,0 +1,158 @@
+# Parachute Protocol
+
+## Introduction
+
+The parachute protocol allows MAVLink control over the behavior of a parachute that is integrated or attached to the drone. Along with this, the parachute protocol also publishes status information for developers or users. 
+
+
+## MAVLink Parachute Implementations
+
+These parachutes have built-in MAVLink support:
+
+- AVSS Parachute for X10
+
+## Message/Enum Summary
+
+| Message                                                                                                              | Description               |
+| ------------------------------------------------------------------------------------------------------------------ | ------------------------- |
+| <span id="PARACHUTE_STATUS"></span>[PARACHUTE_STATUS](../messages/common.md#PARACHUTE_STATUS)                                           | Current status of the parachute. Recommended to publish this at a regular rate.  
```

Comment:
is it recommended to be published at a regular rate, or is that a requirement? I think it's necessary, at least for our system. Also, would be good to specify what the nominal rate should be. Do they send this at 1Hz right now?

Reply:
added nominal rate but i think its recommended but required for ours